appveyor.yml

```diff
@@ -0,0 +1,27 @@
+matrix:
+  fast_finish: true
+
+environment:
+  matrix:
+    - CONDA_PY: 38
+      CONDA_PY_SPEC: ">=3.8,<3.9.0a0"
+      CONDA_INSTALL_LOCN: "C:\\Miniconda36-x64"
+    - CONDA_PY: 37
+      CONDA_PY_SPEC: ">=3.7,<3.8.0a0"
```

Comment:
I think you can just ask conda for `python=3.7`, and it knows that means the highest available 3.7 version. This is how I'm used to using it, but maybe that's not recommended, or maybe I'm missing some corner case where specifying it this way makes a difference.

Reply:
You're probably right, I was following some recent changes in the notebook repo.  That said, I can't get the variables to work at all.  Very frustrating to "debug" Windows solely by way of appveyor results! 😞 
I'll have to play with the settings some more.